<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgTrackInfo</class>
 <widget class="QDialog" name="DlgTrackInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Track Editor</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="lblSong">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Track:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolBox" name="toolbox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>400</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pgTrack">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>409</width>
        <height>454</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Track Information (Click to Expand)</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Title:</string>
           </property>
           <property name="buddy">
            <cstring>txtTrackName</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="3">
          <widget class="QLineEdit" name="txtTrackName"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Artist:</string>
           </property>
           <property name="buddy">
            <cstring>txtArtist</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="3">
          <widget class="QLineEdit" name="txtArtist"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Album:</string>
           </property>
           <property name="buddy">
            <cstring>txtAlbum</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="3">
          <widget class="QLineEdit" name="txtAlbum"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Date:</string>
           </property>
           <property name="buddy">
            <cstring>txtYear</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="3">
          <widget class="QLineEdit" name="txtYear"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Track #:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Genre:</string>
           </property>
           <property name="buddy">
            <cstring>txtGenre</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="1" colspan="3">
          <widget class="QLineEdit" name="txtGenre"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Composer:</string>
           </property>
           <property name="buddy">
            <cstring>txtComposer</cstring>
           </property>
          </widget>
         </item>
         <item row="6" column="1" colspan="3">
          <widget class="QLineEdit" name="txtComposer"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Filename:</string>
           </property>
           <property name="buddy">
            <cstring>txtFilepath</cstring>
           </property>
          </widget>
         </item>
         <item row="7" column="1" colspan="3">
          <widget class="QLineEdit" name="txtFilepath">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Duration:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="txtDuration">
           <property name="text">
            <string>3:00</string>
           </property>
          </widget>
         </item>
         <item row="9" column="2">
          <widget class="QLabel" name="Type">
           <property name="text">
            <string>File Type:</string>
           </property>
          </widget>
         </item>
         <item row="9" column="3">
          <widget class="QLabel" name="txtType">
           <property name="text">
            <string>77</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Comments:</string>
           </property>
           <property name="buddy">
            <cstring>txtComment</cstring>
           </property>
          </widget>
         </item>
         <item row="10" column="1" colspan="3">
          <widget class="QTextEdit" name="txtComment">
           <property name="tabChangesFocus">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="3">
          <widget class="QLineEdit" name="txtTrackNumber"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QPushButton" name="btnReloadFromFile">
           <property name="text">
            <string>Reload track metadata from file.</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pgBpm">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>378</width>
        <height>149</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Track BPM</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="labelBpm">
               <property name="text">
                <string>Track BPM: </string>
               </property>
               <property name="buddy">
                <cstring>spinBpm</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="spinBpm">
               <property name="decimals">
                <number>8</number>
               </property>
               <property name="maximum">
                <double>500.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="bpmDouble">
               <property name="text">
                <string>x2</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bpmHalve">
               <property name="text">
                <string>/2</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="bpmTap">
           <property name="minimumSize">
            <size>
             <width>121</width>
             <height>91</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>123</width>
             <height>91</height>
            </size>
           </property>
           <property name="text">
            <string>Tap to Beat</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Hint: Use the Library Analyze view to run BPM detection.</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>200</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pgCue">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>258</width>
        <height>144</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Track Cuepoints</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="cueTable">
         <column>
          <property name="text">
           <string>Cue Id</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Position</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Hotcue</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Label</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnCueActivate">
           <property name="text">
            <string>Activate Cue</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCueDelete">
           <property name="text">
            <string>Delete Cue</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="btnPrev">
       <property name="text">
        <string>&lt;&lt; &amp;Prev</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnApply">
       <property name="text">
        <string>&amp;Apply</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut">
        <string>Alt+O</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNext">
       <property name="text">
        <string>&amp;Next &gt;&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
